src/api/rest.js

```diff
@@ -11,8 +11,9 @@ import {
 const API_URL = `${process.env.GNOSISDB_URL}/api`
 
 export const requestMarket = async marketAddress =>
-  restFetch(`${API_URL}/markets/${hexWithoutPrefix(marketAddress)}/`)
-    .then(response => normalize(response, marketSchema))
+  restFetch(`${API_URL}/markets/${hexWithoutPrefix(marketAddress)}/`).then(response =>
+    normalize({ ...response, local: false }, marketSchema),
```

Comment:
`local: false`? What is it?

Reply:
@apanizo When we create a market it has property `local`, this means that market is saved locally in redux store and not in gnosisDB. We need some time to wait for gnosisDB to detect it. When it successfully fetched, we change `local` to false